.github/workflows/gradle.yml

```diff
@@ -47,12 +47,27 @@ jobs:
           GRADLE_ENTERPRISE_CACHE_PASSWORD: ${{ secrets.GRADLE_ENTERPRISE_CACHE_PASSWORD }}
         run: |
           [ -f ./setup.sh ] && ./setup.sh || true
+      - name: Build with Gradle only Java 8
+        if: matrix.java == '8'
+        run: |
+          ./gradlew check -x :sql-bom:checkVersionCatalogCompatibility --no-daemon --parallel --continue
```

Comment:
There are two modules which are not published for Java 8. `jooq` and `hibernate-reactive`. 

Without this change to the github actions this will fail: 

```
./gradlew :sql-bom:checkVersionCatalogCompatibility
Type-safe project accessors is an incubating feature.
Project accessors enabled, but root project name not explicitly set for 'buildSrc'. Checking out the project in different folders will impact the generated code and implicitly the buildscript classpath, breaking caching.
> Task :sql-bom:checkVersionCatalogCompatibility FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':sql-bom:checkVersionCatalogCompatibility'.
> Version catalogs are not compatible:
  The following libraries were present in the baseline version but missing from this catalog:
    - micronaut-jooq
    - micronaut-hibernate-reactive
```

Reply:
We've fixed this in another way in other modules with similar issue, which I think is easier for maintenance:

https://github.com/micronaut-projects/micronaut-data/blob/master/data-bom/build.gradle#L15-L19

Reply:
thanks https://github.com/micronaut-projects/micronaut-sql/pull/787/commits/211150a66cb31f16b24896910e0f24f2c0012c5a